src/components/Popover/Popover.tsx

```diff
@@ -12,10 +12,10 @@ import {PreferredPosition, PreferredAlignment} from '../PositionedOverlay';
 import {Portal} from '../Portal';
 import {portal} from '../shared';
 import {useUniqueId} from '../../utilities/unique-id';
-import {CloseSource, Pane, PopoverOverlay, Section} from './components';
+import {PopoverCloseSource, Pane, PopoverOverlay, Section} from './components';
```

Comment:
Name this import/export correctly where it is initially defined, avoids having to rename it in the `Popover/index.ts` file